go.mod

```diff
@@ -1,22 +1,30 @@
 module github.com/binance-chain/go-sdk
 
 require (
-	github.com/binance-chain/bnc-go-amino v0.14.1-binance.1
 	github.com/btcsuite/btcd v0.0.0-20181013004428-67e573d211ac
 	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
+	github.com/coreos/go-iptables v0.4.0
 	github.com/cosmos/go-bip39 v0.0.0-20180819234021-555e2067c45d
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/go-kit/kit v0.8.0 // indirect
 	github.com/go-logfmt/logfmt v0.3.0 // indirect
 	github.com/gogo/protobuf v1.1.1 // indirect
-	github.com/golang/protobuf v1.2.0 // indirect
 	github.com/gorilla/websocket v1.4.0
 	github.com/pkg/errors v0.8.0
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/prometheus/client_golang v0.9.2 // indirect
+	github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
+	github.com/rs/cors v1.6.0 // indirect
 	github.com/stretchr/testify v1.2.2
+	github.com/syndtr/goleveldb v1.0.0 // indirect
 	github.com/tendermint/btcd v0.0.0-20180816174608-e5840949ff4f
 	github.com/tendermint/ed25519 v0.0.0-20171027050219-d8387025d2b9 // indirect
+	github.com/tendermint/go-amino v0.14.1
```

Comment:
Safe to use this? bnc-go-amino binary encoding/decoding is a bit different.

Reply:
delete it.

Reply:
The  bottom of go.mod there is a replace.  Finally it will use bnc-go-amino. This is also emphasize in the doc